trashies.go

```diff
@@ -57,24 +57,45 @@ func DeletePerson(w http.ResponseWriter, r *http.Request) {
 }
 
 func launch(w http.ResponseWriter, r *http.Request) {
-	t, _ := template.ParseFiles("index.html", "footer.html", "header.html")
-	t.ExecuteTemplate(w,"index", nil)
+	t, _ := template.ParseFiles("index.html", "templates/footer.html", "templates/navbar.html", "templates/header.html", "templates/jsimports.html")
+	t.ExecuteTemplate(w, "index", nil)
+}
+
+func join(w http.ResponseWriter, r *http.Request) {
+	t, _ := template.ParseFiles("join.html", "templates/footer.html", "templates/navbar.html", "templates/header.html", "templates/jsimports.html")
+	t.ExecuteTemplate(w, "join", nil)
+}
+
+func waiting(w http.ResponseWriter, r *http.Request) {
+	t, _ := template.ParseFiles("waiting.html", "templates/footer.html", "templates/navbar.html", "templates/header.html", "templates/jsimports.html")
+	t.ExecuteTemplate(w, "waiting", nil)
+}
+
+func voting(w http.ResponseWriter, r *http.Request) {
+	t, _ := template.ParseFiles("voting.html", "templates/footer.html", "templates/navbar.html", "templates/header.html", "templates/jsimports.html")
+	t.ExecuteTemplate(w, "voting", nil)
 }
 
 func main() {
 
 	cssHandler := http.FileServer(http.Dir("./css/"))
 	jsHandler := http.FileServer(http.Dir("./js/"))
+	imageHandler := http.FileServer(http.Dir("./images/"))
```

Comment:
So I haven't jumped into go quite yet, but I'm guessing main() is the main called function? And all of these handling just get called and fills in the pictures defined by the html?

Reply:
these are taking our local directories of css, js, and images and hosting them on the server so that the browser can actually see them. when we were working with pure html it was enough to just have them in the same folder but since everything is being served up by go, go needs to know where the files are.

Reply:
Also yes, main is the entry point into the application.